Error: The `form` element was not requested.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label. Note: If the checkbox control is not nested in its `label`, wrapping the two elements within another element (such as a `div`) is preferable. Having a shared parent element gives designers more control over the eventual layout of the checkbox group than a `br` does.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
Error: `aria-invalid="true"` is used to accessibly indicate fields that have errors.
Error: Color alone should not be used to convey errors.
Error: The error text should not be hidden when the component has an error.
diff --git a/output/Checkbox Group with Validation Error/1/d4993104-5bd8-49ca-b7cf-5df088556935.html b/output/Checkbox Group with Validation Error/1/d4993104-5bd8-49ca-b7cf-5df088556935.html
index 8070eda..ed94dcd 100644
--- a/output/Checkbox Group with Validation Error/1/d4993104-5bd8-49ca-b7cf-5df088556935.html	
+++ b/output/Checkbox Group with Validation Error/1/d4993104-5bd8-49ca-b7cf-5df088556935.html	
@@ -1,12 +1,16 @@
-<form>
-  <fieldset>
-    <legend>What fruits do you like?</legend>
-    <input type="checkbox" id="bananas" name="fruits" value="bananas">
-    <label for="bananas">Bananas</label><br>
-    <input type="checkbox" id="oranges" name="fruits" value="oranges">
-    <label for="oranges">Oranges</label><br>
-    <input type="checkbox" id="apples" name="fruits" value="apples">
-    <label for="apples">Apples</label><br>
-    <span id="error" style="color: red; display: none;">You must choose one or more fruits</span>
-  </fieldset>
-</form>
\ No newline at end of file
+<fieldset aria-describedby="error">
+  <legend>What fruits do you like?</legend>
+  <span id="error">You must choose one or more fruits</span>
+  <div>
+    <input type="checkbox" id="bananas" name="fruits[]" value="bananas" aria-invalid="true">
+    <label for="bananas">Bananas</label>
+  </div>
+  <div>
+    <input type="checkbox" id="oranges" name="fruits[]" value="oranges" aria-invalid="true">
+    <label for="oranges">Oranges</label>
+  </div>
+  <div>
+    <input type="checkbox" id="apples" name="fruits[]" value="apples" aria-invalid="true">
+    <label for="apples">Apples</label>
+  </div>
+</fieldset>
\ No newline at end of file
